cip.go

```diff
@@ -72,6 +72,16 @@ func main() {
 		"version",
 		false,
 		"print version")
+	createManifestPtr := flag.Bool(
+		"create-manifest",
+		false,
+		"generating manifest file from all images in destination"+
+			" repositories instead of promoting images")
+	createMfestOutputPtr := flag.String(
```

Comment:
Can we merge this option with the createManifestPtr option?

If I say `-create-manifest=STDOUT` it should print to STDOUT.
If I say `-create-manifest=<any-other-nonzero-path>` it should save to that path.
If the `-create-manifest` flag is not specified, it's the same as `false` (default).

I think `flag.StringVar` should work for this?

Reply:
Hmm maybe it would be more elegant if we could have this behavior:

Passing `-create-manifest` alone should print to STDOUT.
Passing `-create-manifest=<path>` will save to that path.

Reply:
As discussed, I completely removed option to save to the path.